app/src/main/java/app/gamenative/service/gog/GOGConstants.kt

```diff
@@ -0,0 +1,60 @@
+package app.gamenative.service.gog
+
+import app.gamenative.PrefManager
+import java.io.File
+import java.nio.file.Paths
+import timber.log.Timber
+
+/**
+ * Constants for GOG integration
+ */
+object GOGConstants {
+    // GOG API URLs
+    const val GOG_BASE_API_URL = "https://api.gog.com"
+    const val GOG_AUTH_URL = "https://auth.gog.com"
+    const val GOG_EMBED_URL = "https://embed.gog.com"
+    const val GOG_GAMESDB_URL = "https://gamesdb.gog.com"
+
+    // GOG Client ID for authentication
+    const val GOG_CLIENT_ID = "46899977096215655"
+
+    // GOG uses a standard redirect URI that we can intercept
+    const val GOG_REDIRECT_URI = "https://embed.gog.com/on_login_success?origin=client"
+
+    // GOG OAuth authorization URL with redirect
+    const val GOG_AUTH_LOGIN_URL = "https://auth.gog.com/auth?client_id=$GOG_CLIENT_ID&redirect_uri=$GOG_REDIRECT_URI&response_type=code&layout=client2"
+
+    // GOG paths - following Steam's structure pattern
+    private const val INTERNAL_BASE_PATH = "/data/data/app.gamenative/files"
```

Comment:
https://github.com/utkarshdalal/GameNative/pull/371/files

It should be GOG under /data/data/app.gamenative, not /data/data/app.gamenative/files